homeassistant/components/wiffi/sensor.py

```diff
@@ -97,7 +103,18 @@ def _update_value_callback(self, device, metric):
         self._unit_of_measurement = UOM_MAP.get(
             metric.unit_of_measurement, metric.unit_of_measurement
         )
+
+        if self._is_metered_entity():
+            # try to detect reset for metered entities
+            if (
+                metric.value is not None
+                and self._value is not None
+                and metric.value < self._value
+            ):
+                self._attr_last_reset = utcnow()
```

Comment:
This means it's only evaluated/set on the first update.

So what if I restart Home Assistant in between?